migrations/202107061359_create_webhooks.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE IF NOT EXISTS webhooks (
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(60) NOT NULL,
+  type SMALLINT NOT NULL,
+  status SMALLINT NOT NULL,
+  url VARCHAR(256) NOT NULL,
```

Comment:
same for URL, I'd also suggest using 300 to align with the other tables and update the action validation as well

Reply:
I'll change it to a `TEXT` type because it's a template and it may be larger than we expect (with route or GET parameters defined from the Go template). I've adapted the validation to handle the parsing before calling URL validation